cypress/integration/manual-aggregated-sync.spec.js

```diff
@@ -1,27 +1,137 @@
-import { dataTest } from "../support/utils";
+import ManualAggregateSyncPageObject from "../pageobjects/ManualAggregateSyncPageObject";
 
+/**
+ * Database: d2-docker-eyeseetea-2-30-datasync-sender
+ */
 context("Manual aggregated sync", function() {
+    const page = new ManualAggregateSyncPageObject(cy);
+
+    const anyOrgUnit = "Ghana";
+    const anyInstance = "Y5QsHDoD4I0";
+    const anyDataset = "Malaria annual data";
+
     beforeEach(() => {
-        cy.login("admin");
-        cy.visit("/#/sync/aggregated");
-        cy.get(dataTest("headerbar-title")).contains("MetaData Synchronization");
-    });
-
-    it("has the correct title", function() {
-        cy.get(dataTest("page-header-title")).contains("Aggregated Data Synchronization");
-    });
-
-    /**
-     * Database: d2-docker-eyeseetea-2-30-datasync-sender
-     */
-    it("syncs correctly malaria annual data", function() {
-        /**cy.get('[data-test="search"] > div > [aria-invalid="false"]').type("Malaria annual data");
-        cy.get(dataTest("TableCell-data-table-row-0-column-displayname")).contains(
-            "Malaria annual data"
-        );*/
-        cy.get(dataTest("Checkbox-data-table-row-0")).click();
-        cy.get(
-            '[data-test="objects-table-action-button"] > :nth-child(1) > [focusable="false"]'
-        ).click();
+        page.open();
+    });
+
+    it("should have the correct title", function() {
+        page.title.contains("Aggregated Data Synchronization");
+    });
+
+    it("should syncs correctly malaria annual data", function() {
+        page.search(anyDataset)
+            .selectRow(anyDataset)
+            .openSyncDialog()
+
+            .selectOrgUnit(anyOrgUnit)
+            .next()
+
+            .selectAllPeriods()
+            .next()
+
+            .selectAllAttributesCategoryOptions()
+            .next()
+
+            .selectReceiverInstance(anyInstance)
+            .synchronize()
+
+            .syncResults.contains("Success");
+
+        page.closeSyncResultsDialog();
+    });
+
+    it("should show the org unit step error if user try click on next without selecting the org unit", function() {
```

Comment:
minor: from ES6 we tend to write lambdas as arrow funcions `() => {}`

Reply:
Ok, I will do it

Reply:
Done!